---
title: "Robert's Useful Web Links"
date: "`r Sys.Date()`"
output: distill::distill_article
---

[paperpile](https://paperpile.com/app)


# Rheumatology learning resources

[Rheum2Learn](https://www.rheumatology.org/Learning-Center/Educational-Activities/Rheum2Learn)

# Peer Reviewed Papers

[ConnectedPapers](https://www.connectedpapers.com/)

[PubMed](https://pubmed-ncbi-nlm-nih-gov.proxy.library.vanderbilt.edu/?otool=vandylib)

[GoogleScholar](https://scholar.google.com/)

[VUMC library](https://www.library.vanderbilt.edu/biomedical/)

[Nature Reviews: Immunology](https://www-nature-com.proxy.library.vanderbilt.edu/nri/)

[Nature Reviews: Rheumatology](https://www-nature-com.proxy.library.vanderbilt.edu/nrrheum/)

# Books

[Hochberg Rheumatology](https://www-clinicalkey-com.proxy.library.vanderbilt.edu/#!/browse/book/3-s2.0-C20191000873)


# Clinical Resources

[UpToDate](https://www-uptodate-com.ckmproxy.vumc.org/contents/search)


# Online-only Resources

[Immunopedia](https://www.immunopaedia.org.za/)

[T cell mediated diseases per above](https://www.immunopaedia.org.za/immunology/advanced/9-t-cell-mediated-autoimmune-diseases/)

[HCDM](https://hcdm.org): human cell differentiation molecules society (names CD's)



# Molecular and Cellular data

TODO
[twitter thread of resources](https://twitter.com/tangming2005/status/1666437518907133954?t=AVmFFDJNjvUyccTAdJtCkA)

[GWAS atlas](https://atlas.ctglab.nl)

[UKBB proteomics](https://metabolomips.org/ukbbpgwas/)

[ABseq](https://apps.embl.de/abseqapp/): A resource of gene and surface protein expression in human bone marrow

[CRUSTY](https://crusty.humanitas.it/): ClusteRing UnsuperviSed meThods for high dimensional cYtometry data

[tabula sapiens](https://tabula-sapiens-portal.ds.czbiohub.org/): first-draft human cell atlas of nearly 500,000 cells from 24 organs of 15 normal human subjects.

[CELLxGENE](https://cellxgene.cziscience.com/): largest corpus of integrated single-cell data

[enrichr-KG](https://maayanlab.cloud/enrichr-kg): gene set enrichment analysis

[stringDB](https://string-db.org/): protein-protein interaction network based on multiple data sources (including text mining)

[WebGestalt](https://www.webgestalt.org/) pathway analysis

[bioGPS](http://biogps.org/#goto=welcome): a "bioGPS" - maybe good for looking up genes?

[a ubiquitin database](http://uucd.biocuckoo.org/): todo - explain more what this is

[Compartments](https://compartments.jensenlab.org/Downloads): subcellular localization database

[Antigen Explorer](https://antigen.princeton.edu/)Uses expression data from 33 tumor types and 34 normal tissues to explore antigen combinations

[UCSD genome browser](https://genome.ucsc.edu/) many tools, LiftOver switches SNP's between assemblies

[genebass](https://app.genebass.org/): single-variant based, whole-exome association statistics on 4,529 traits and 394,841 patients from UKBB.

[AZphewas](https://azphewas.com/): exome-sequencing based gene-trait associations from UKBB

[HBFP](https://bmbl.bmi.osumc.edu/HBFP/index.php/Home/Index/index.html): human body fluid proteome

[omicsPred](https://www.omicspred.org/): an atlas of SNP effects (via ridge regression I believe) on mRNA levels, protein levels, and metabolite levels.

[causalDB](http://www.mulinlab.org/causaldb/index.html): at atlas of snp-trait associations with an emphasis on fine-mapping to identify causal SNP's

[MetaXcan](https://github.com/hakyimlab/MetaXcan)

[Signaling Pathways Project](http://www.signalingpathways.org/index.jsf)

[SCALLOP](http://www.scallop-consortium.com/): Systematic and Combined AnaLysis of Olink Proteins

[IMGT](https://www.imgt.org/): International Immunogenetics

[TCRdb (broken)](http://bioinfo.life.hust.edu.cn/TCRdb/)

[TCR3d](https://tcr3d.ibbr.umd.edu/): T cell receptor structural repertoire database

[OpenAIRE Explore](https://explore.openaire.eu/)

[immune cell atlas](https://www.tissueimmunecellatlas.org/)

[human cell atlas](https://data.humancellatlas.org/)

[GTEx](https://gtexportal.org/home/): GTEx home

[6GTEx - single cell resources](https://gtexportal.org/home/singleCellOverviewPage)

[IGSR](https://www.internationalgenome.org/home): The International Genome Sample Resource

[ImmunexUT](https://www.immunexut.org/top): Dynamic landscape of immune cell-specific gene regulation in immune-mediated diseases

# Population Genomics

GnomAD

[human gene mutation database](https://www.hgmd.cf.ac.uk/)

Clinvar


# Clinical Data

[UKBB](https://bbams.ndph.ox.ac.uk/ams/researcher_home.jsp)

[SD-discover](https://sd.app.vumc.org/sd-discover/)

[VUMC clinical data resources](https://www.vumc.org/vclic/data-resources)




# compute resources

[ACCRE virtual sessions](https://portal.accre.vanderbilt.edu/pun/sys/dashboard/batch_connect/sessions)

[CellTypist](https://www.celltypist.org/): Automated cell type annotation for scRNA-seq datasets

[NCI Analysis Tools](https://analysistools.cancer.gov/)

[Robetta](https://robetta.bakerlab.org/): A protein structure prediction service from the Baker lab

[GrayLab @ JHU](https://r2.graylab.jhu.edu/apps/index)


# Software

[VDJmatch](https://github.com/antigenomics/vdjmatch)

[muon](https://gtca.github.io/muon/): multimodal omics in Python

[anvio](https://anvio.org/): An open-source, community-driven analysis and visualization platform for microbial 'omics.


# People

[VCI](https://www.vumc.org/vci/vci-faculty): Vanderbilt Center for Immunology
